scene/resources/texture.cpp

```diff
@@ -370,6 +371,15 @@ void PortableCompressedTexture2D::_set_data(const Vector<uint8_t> &p_data) {
 				uint32_t mipsize = decode_uint32(data);
 				data += 4;
 				data_size -= 4;
+
+				// Skip possible Godot png tag "PNG " before pure PNG data
```

Comment:
[Godot own PNG prefix](https://github.com/godotengine/godot/blob/ffd32a244b43ff58c13819c2debf8cf3b58ecbdc/drivers/png/image_loader_png.cpp#L80) must been skipped

Reply:
This comment should also be correctly formatted

Reply:
Done)